redbot/core/bot.py

```diff
@@ -289,6 +289,18 @@ def unload_extension(self, name):
             if pkg_name.startswith("redbot.cogs"):
                 del sys.modules["redbot.cogs"].__dict__[name]
 
+    def add_cog(self, cog):
+        for attr in dir(cog):
+            _attr = getattr(cog, attr)
+            if isinstance(_attr, discord.ext.commands.Command) and not isinstance(
+                _attr, commands.Command
+            ):
+                raise RuntimeError(
+                    f"{cog.__class__.__name} from {cog.__module__},"
```

Comment:
`cog.__class__.__name` should be `cog.__class__.__name__` otherwise it raises an attribute error.